### ISSUE TITLE
[Feature Request] - Suggestion for much better sync results

### ISSUE THREAD

**Issue by prorun26** [Lybron/health-auto-export#29]
At the moment, the current values ​​of the current day are synchronized at the set interval.

Unfortunately, this is very error-prone, as 11:59 p.m. cannot always be synchronized and therefore very often the last correct values ​​cannot be synchronized.

Wouldn't it be much more effective to synchronize the last/highest value of the previous day once a day?

This way, you would always get the guaranteed last/highest value of yesterday on the following day with just one single synchronization.

**Comment by prorun26** [comment]
A good option would perhaps be a quantity "yesterday" in the sync cadence section

**Comment by Lybron** [comment]
Yep, I think that would likely be the simplest solution, and would follow the pattern used for the other automation types. I can add that 👍

**Comment by leehadassin** [comment]
I would also like this - I’m really after the long term daily data for trending rather than the today data (the “today” data is glanceable on my watch already)

**Comment by Lybron** [comment]
There's the option to select the "Yesterday" time period for Home Assistant automations as of v8.2.7. 

Feel free to reopen and follow up if  there are any issues or you'd like any further changes.
